app/assets/javascripts/views/restrooms/restrooms.js

```diff
@@ -0,0 +1,49 @@
+/*
+Place all the behaviors and hooks related to the matching controller here.
+All this logic will automatically be available in application.js.
+*/
+$(function(){
```

Comment:
Is there something more rails or a coffeescript querystring class to replace this with?